### PR TITLE
docs: add env, job, svc override cmds

### PR DESCRIPTION
**Pull request by efekarakus**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/docs/commands/env-override.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# copilot env override
+```console
+$ copilot env override
+```
+
+## What does it do?
+Scaffold Infrastructure as Code (IaC) extension files for environments.
+The generated files allow you to extend and override the Copilot-generated AWS CloudFormation template.
+You can edit the files to change existing resource properties, delete
```
Comment (huanjani) [inline site/content/docs/commands/env-override.md]:
```suggestion
You can edit the files to change existing resource properties, and delete
```

File: site/content/docs/commands/job-override.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# copilot job override
+```console
+$ copilot job override
+```
+
+## What does it do?
+Scaffold Infrastructure as Code (IaC) extension files for a job.
+The generated files allow you to extend and override the Copilot-generated AWS CloudFormation template.
+You can edit the files to change existing resource properties, delete
```
Comment (huanjani) [inline site/content/docs/commands/job-override.md]:
```suggestion
You can edit the files to change existing resource properties, and delete
```

File: site/content/docs/commands/svc-override.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# svc override
+```console
+$ copilot svc override
+```
+
+## What does it do?
+
+Scaffold Infrastructure as Code (IaC) extension files for a service.
+The generated files allow you to extend and override the Copilot-generated AWS CloudFormation template.
+You can edit the files to change existing resource properties, delete
```
Comment (huanjani) [inline site/content/docs/commands/svc-override.md]:
```suggestion
You can edit the files to change existing resource properties, and delete
```

File: site/content/docs/commands/env-override.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+Create a new Cloud Development Kit application to override environment templates.
+
+```console
+$ copilot env override --tool cdk
+```
+
+## What does it look like?
+
+![env-override](https://user-images.githubusercontent.com/879348/227585768-44d5d91f-11d5-4d4b-a5fa-12bb5239710f.gif)
```
Comment (dannyrandall) [inline site/content/docs/commands/env-override.md:40]: :meow_wow:

File: site/content/docs/commands/env-override.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+To learn more check out the guides for overriding with [YAML Patches](../developing/overrides/yamlpatch.md) and the
+[AWS Cloud Development Kit](../developing/overrides/cdk.md).
+
+## What are the flags?
+
+```console
+  -a, --app string            Name of the application.
+      --cdk-language string   Optional. The Cloud Development Kit language. (default "typescript")
+  -h, --help                  help for override
```
Comment (dannyrandall) [inline site/content/docs/commands/env-override.md]: we should prob capitalize this...
```suggestion
  -h, --help                  Help for override
```

**Review by dannyrandall — APPROVED** [review]

**Review by huanjani — APPROVED** [review]
:shipit:

**Pull request merged**